commercetools-models/src/main/java/io/sphere/sdk/customers/commands/updateactions/SetCustomerGroup.java

```diff
@@ -22,19 +23,20 @@
     @Nullable
     private final ResourceIdentifier<CustomerGroup> customerGroup;
 
-    private SetCustomerGroup(final Referenceable<CustomerGroup> customerGroup) {
+    private SetCustomerGroup(final ResourceIdentifier<CustomerGroup> customerGroup) {
         super("setCustomerGroup");
-        this.customerGroup = Optional.ofNullable(customerGroup)
-                .map(x -> x.toReference().filled(null))
-                .orElse(null);
+        this.customerGroup = customerGroup;
     }
 
-    public static SetCustomerGroup of(@Nullable final Referenceable<CustomerGroup> customerGroup) {
-        return new SetCustomerGroup(customerGroup);
+    public static SetCustomerGroup of(@Nullable final ResourceIdentifiable<CustomerGroup> customerGroup) {
```

Comment:
I found the same method and action in the cart, for the same purpose of the ticket could make sense to do it for the SetCustomerGroup in the Cart action @ashishhk ? It would be difficult to pass a Referenceable in the of method, isn't it?
this is the part of the code 
    `public static SetCustomerGroup of(@Nullable final Referenceable<CustomerGroup> customerGroup) {
        final ResourceIdentifier<CustomerGroup> resourceIdentifier = Optional.ofNullable(customerGroup)
                .map(ResourceIdentifiable::toResourceIdentifier)
                .orElse(null);
        return new SetCustomerGroup(resourceIdentifier);
    }
`

Reply:
a Referenceable extends ResourceIdentifiable. So totally valid